src/LmdbCache.ts

```diff
@@ -100,4 +101,63 @@ export class LmdbCache<V = any> implements SortKeyCache<V> {
   storage<S>(): S {
     return this.db as S;
   }
+
+  async prune(entriesStored: number = 1): Promise<PruneStats> {
```

Comment:
`yarn lint` please :-)

Reply:
I'm used to `;` at the end of line :-( but not sure if `yarn lint` will handle this...